lib/pbench/test/unit/server/test_sync.py

```diff
@@ -52,6 +52,19 @@ def test_error_new(self, make_logger, more_datasets):
             "UPLOAD": {"state": "FAILED", "message": "this is an error"}
         }
 
+    def test_error_long(self, make_logger, more_datasets):
+        """Test handling of long error messages"""
```

Comment:
Didn't want to fix it this time, either, eh?  🤣

Reply:
I was momentarily tempted; but this is a cherry pick to b0.72, so changing the original code isn't really kosher.